### PR TITLE
fix: trigger timemachine in global decorator, and leave it ticking

### LINKED ISSUE #3896: Storybook knobs aren't working
**Describe the bug**
inputProvider service toggling isnt working

**To Reproduce**

1. Go to https://bbc.github.io/simorgh/?path=/story/containers-header--default
2. Flick through other services
3. See no change

NB: After investiating, it appears storybook `addon-knobs` are entirely broken in simorgh. They show, but changing them have no affect

**Expected behavior**
It changes for each service.

**Screenshots**

**Testing notes**

- [ ] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by mykeels**
Resolves #3896

**Overall change:** Trigger [timemachine](https://github.com/schickling/timemachine) in global decorator, for stories that display timestamps, and leave the timemachine ticking, so knobs _(which depend on setTimeout)_ can work._

**Code changes:**

- _Remove timemachine refs from .storybook/config.js._
- _Add decorator to ArticleTimestamp stories, so timemachine is configured on mount, and reset on unmount._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

**Review by samora — APPROVED** [review]
Nice work.

**Review by chris-hinds — COMMENTED** [review]
Are we not using timestamps anywhere else in storybook? Will this not lead to us having inconsistencies in chrimaticqa snapshots again because of the time change?

File: src/app/lib/utilities/moment/index.stories.jsx [hunk 1]
```diff
@@ -89,100 +89,664 @@ const editorialWhitelist = [
...
-  { what: 'LTS', method: locale => moment(A_DATE).locale(locale).format('LTS') },
-  { what: 'l', method: locale => moment(A_DATE).locale(locale).format('l') },
-  { what: 'L', method: locale => moment(A_DATE).locale(locale).format('L') },
-  { what: 'll', method: locale => moment(A_DATE).locale(locale).format('ll') },
-  { what: 'LL', method: locale => moment(A_DATE).locale(locale).format('LL') },
-  { what: 'lll', method: locale => moment(A_DATE).locale(locale).format('lll') },
-  { what: 'LLL', method: locale => moment(A_DATE).locale(locale).format('LLL') },
-  { what: 'llll', method: locale => moment(A_DATE).locale(locale).format('llll') },
-  { what: 'LLLL', method: locale => moment(A_DATE).locale(locale).format('LLLL') },
+  {
```
Comment (dr3) [inline src/app/lib/utilities/moment/index.stories.jsx:91]: This file has `/* eslint-disable prettier/prettier */` on line 70 to ensure that linters dont correct this file, making it far far too long. Perhaps your editor did this?

Can you reset this please, for this case, fewer longer lines is much much easier to read than 700 shorter ones :)

File: .storybook/config.js [hunk 2]
```diff
@@ -48,6 +42,8 @@ addDecorator(story => (
   /* eslint-enable react/jsx-filename-extension */
 ));
 
+addDecorator(story => <WithTimemachine>{story()}</WithTimemachine>);
```
Comment (dr3) [inline .storybook/config.js:45]: This Pr is titled `restrict timemachine to ArticleTimestamp stories`, is it doing that? This is a global decorator

**Review by dr3 — CHANGES_REQUESTED** [review]

File: .storybook/config.js (hunk 2 above)
Comment (mykeels) [reply .storybook/config.js:45]: We initially started out as that, before realising a global decorator was possible. I'll change the PR title.

**Comment by hotinglok** [comment]
Updated here: https://github.com/bbc/simorgh/pull/4189

**Pull request closed without merging**